src/eos-metrics-instrumentation.c

```diff
@@ -78,6 +90,96 @@ static gint64 start_time;
 
 static EinsPersistentTally *persistent_tally;
 
+static gboolean
+get_os_version (char **name_out,
```

Comment:
I prefer not to use them too, and I don't use them in my own code, but there was quite heavy opposition (mainly from Igalia) when I proposed dropping them, back when we started writing eos-shell and eos-sdk. For better or worse, they're part of our style guide.